tests/examples/example10-schDisplayLabel.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { expect, test } from "bun:test"
+import { getTestFixture } from "../fixtures/get-test-fixture"
+
+test("Should not render any schematic components", async () => {
+  const { circuit } = getTestFixture()
+
+  circuit.add(
+    <board width={20} height={20}>
+      <resistor
+        name="R1"
+        footprint="0603"
+        schX={8}
+        schY={-1}
+        pcbX={0}
+        resistance="10k"
+      />
+      <resistor
+        name="R2"
+        footprint="0603"
+        schX={0}
+        schY={-1}
+        pcbX={4}
+        resistance="10k"
+      />
+      <capacitor
+        name="C1"
+        footprint="0603"
+        schX={4}
+        schY={1}
+        pcbX={-4}
+        capacitance="10k"
+      />
+      <capacitor
+        name="C2"
+        footprint="0603"
+        schX={2}
+        schY={0}
+        pcbX={-8}
+        capacitance="10k"
+      />
+
+      <trace from={".R2 > .pin2"} to={".C1 > .pin1"} />
+      <trace schDisplayLabel="U2" path={[".C1 > .pin2", ".C2 > .pin2"]} />
```

Comment:
I find it makes everything easier for people to understand when the example are a bit closer to what would happen in real-world usage. In this case, it would be strange to name a connection the same name as a chip "U1" some possible names:

- SDA_1
- SCL
- I2C_DATA
- MOSI
- DATA1
- D0
- R1_1
- R1_2
- C1_POS
 
```suggestion
      <trace schDisplayLabel="SDA_1" path={[".C1 > .pin2", ".C2 > .pin2"]} />
```